unit-tests-java8/src/test/java/org/eclipse/collections/test/list/mutable/UnmodifiableMutableListTestCase.java

```diff
@@ -15,24 +15,25 @@
 import org.eclipse.collections.impl.block.factory.Comparators;
 import org.eclipse.collections.test.UnmodifiableMutableCollectionTestCase;
 import org.eclipse.collections.test.list.UnmodifiableListTestCase;
-import org.junit.Assert;
 import org.junit.Test;
 
+import static org.junit.Assert.assertThrows;
+
 public interface UnmodifiableMutableListTestCase extends UnmodifiableMutableCollectionTestCase, UnmodifiableListTestCase, MutableListTestCase
 {
     @Override
-    @Test(expected = UnsupportedOperationException.class)
+    @Test
     default void MutableList_sortThis()
     {
-        this.newWith(5, 1, 4, 2, 3).sortThis();
+        assertThrows(UnsupportedOperationException.class, () -> this.newWith(5, 1, 4, 2, 3).sortThis());
     }
 
     @Override
-    @Test(expected = UnsupportedOperationException.class)
+    @Test
     default void MutableList_shuffleThis()
     {
-        this.newWith(5, 1, 4, 2, 3).shuffleThis();
-        this.newWith(5, 1, 4, 2, 3).shuffleThis(new Random(8));
+        assertThrows(UnsupportedOperationException.class, () -> this.newWith(5, 1, 4, 2, 3).shuffleThis());
+        assertThrows(UnsupportedOperationException.class, () -> this.newWith(5, 1, 4, 2, 3).shuffleThis(new Random(8)));
```

Comment:
This is a perfect example of why @Test(expected) should never be used. Thank you for getting rid of these.